### ISSUE TITLE
Publishing SNAPSHOT releases to oss-snapshot-local

### ISSUE THREAD

**Issue by JLLeitschuh** [bintray/gradle-bintray-plugin#191]
According to this document:
https://www.jfrog.com/confluence/display/RTF/Deploying+Snapshots+to+oss.jfrog.org
I should be able to publish my snapshot builds to jfrog bintray.
I asked and have been granted permission to have my artifacts and snapshots hosted on jfrog.
This is my project:
https://bintray.com/jlleitschuh/maven-artifacts/kotlin-guiced

I'm trying to publish a snapshot `0.0.3-SNAPSHOT` to `oss-snapshot-local` using this plugin.
I'm not seeing any documentation about how to do so.

When I try with the current config I get an error:
```
* What went wrong:
Execution failed for task ':kotlin-guiced-core:bintrayUpload'.
> Could not create version '0.0.3-SNAPSHOT': HTTP/1.1 400 Bad Request [message:Unable to create version 0.0.3-SNAPSHOT at 'jlleitschuh/maven-artifacts/kotlin-guiced']
```

Is there some different configuration I need to support this?
This is the code I'm using to configure publishing to bintray:
https://github.com/JLLeitschuh/kotlin-guiced/blob/master/kotlin-guiced.gradle.kts#L38-L51

**Comment by sebersole** [comment]
bump.  is this plugin intended to allow publishing snapshots to OJO?

https://stackoverflow.com/questions/47872069/confused-about-process-to-publish-snapshots-to-bintray

**Comment by snowe2010** [comment]
no, you need to use https://www.jfrog.com/confluence/display/RTF/Gradle+Artifactory+Plugin instead.
